docs/Community/Governance.md

```diff
@@ -0,0 +1,137 @@
+==========================
+
+Like all documentation, this is a living document, a work in progress, representing
+approximately how the community operates and the principles which infuse our
+culture today.
+
+It should be edited and revised from time to time.
+
+Core Principles
+---------------
+*NOTE(Aeva): Suggestion of principles only, but what ever they are, they should be articulated here*
+
+The Open Enclave SDK community adheres to the principles of the Four Opens:
+
+* **Open Source**: All project code and dependencies are open source; see contribution guidelines below.
+* **Open Design**: Project planning and design discussions happen in transparent, collaborative, and accessible ways.
+* **Open Development**: All ideas and contributions are welcome and accepted based on merit and alignment with project goals.
+* **Open Community**: We are welcoming and respectful of everyone.
+
+
+Code of Conduct
+---------------
+
+The Open Enclave SDK community abides by the
+[OE SDK Code of Conduct](conduct/CodeOfConduct.md). Here is an
+excerpt:
+
+*In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and our
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and
+expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.*
+
+As a member of the OE SDK project, you represent the project and your fellow
+contributors. We value our community trememdously and want to build and maintain
+a friendly, inclusive, collaborative environment for all our contributors and
+users.
+
+Community Governance Structure
+------------------------------
+
+*Note(Aeva) section added*
+
+The Open Enclave SDK community uses a
+[concensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+approach to governance at all levels. To support a growing project, we have
+adopted a loose heirarchy wherein domain-specific decisions are handled within
+each domain, or Special Interest Group (**SIG**), cross-domain work is enabled
+through Working Groups (**WG**), and a Community Governance Committee (**CGC**)
+meets to review the overall structure, ensuring the health of the project.
+
+Each group (SIGs, WGs, and CGC) maintain documentation in this repository.
+Creating a new SIG or WG requires the approval of the CGC. SIGs are generally
+responsible for code, whereas a WG is not.
+
+SIG leaders are responsible for the code within their domain, and should be
+considered authoritative for that subsection of the project's code base insofar
+as code reviews and architectural discussions. When a change affects code owned
+by multiple SIGs, it should be coordinated with the respective SIG leads.
+Cross-SIG efforts can be coordinated by establishing a Working Group (WG), which
```

Comment:
Q: Is a WG needed for every cross-SIG effort? (I think no) so would be helpful to know when extra process is needed vs not needed.  (Lightweight is good so spinning up a WG for one PR would be overkill.)
Right now, I'm hesitant to add "WG" discussion in here until there's actually shown to be a need for one, but open to discussion.

Reply:
I thought about omitting WGs from the doc, as they do feel a little process-heavy right now, but on the other hand, I anticipate a need for work that spans several SIGs pretty soon after we adopt a structure that only has SIGs and Committees.

I could break the WG component into a separate PR so that it can sit in the backlog until we need it?

Reply:
I like that idea.  I also have no objection to including it now if others think it's good to document it even if we don't need it yet.

Reply:
I'm in favour of keeping it there, since the description is quite lightweight. It's possible that as the first WG is formed, this may get refined, but that can happen in due course.

Reply:
:+1: